wrappers/azurekeyvault/azurekeyvault.go

```diff
@@ -180,7 +193,11 @@ func (v *Wrapper) SetConfig(ctx context.Context, opt ...wrapping.Option) (*wrapp
 			if keyInfo.Key == nil {
 				return nil, errors.New("no key information returned")
 			}
-			v.currentKeyId.Store(ParseKeyVersion(to.String((*string)(keyInfo.Key.KID))))
+			keyID := ""
```

Comment:
didn't the previous code accomplish the same thing given that to.String(...)
returns an empty string if it's given a ni?

Reply:
Yup, the goal is to remove autorest based libraries as they are deprecated.